lib/widgets/content.dart

```diff
@@ -1351,8 +1360,5 @@ InlineSpan _errorUnimplemented(UnimplementedNode node) {
   }
 }
 
-const errorStyle = TextStyle(
-  fontSize: kBaseFontSize, fontWeight: FontWeight.bold, color: Colors.red);
-
 final errorCodeStyle = kMonospaceTextStyle
   .merge(const TextStyle(fontSize: kBaseFontSize, color: Colors.red));
```

Comment:
Maybe move this at the same time? Seems like we'll need to move it too for dark mode; and in any case it's closely related to `errorStyle`, so it seems like both of them are easier to understand if they're defined next to each other.